resources/views/users/reserved/show_details.blade.php

```diff
@@ -12,7 +12,7 @@
             <div class="category">
                 <h2 class="display-5">My Schedule</h2>
                   <h3 class="fs-3 ms-5">Chat Room</h3>
-                  @forelse($user->joinMeetings as $meeting)
+                  @forelse($user->joinMeetings()->where('date', '>=', today())->where('start_at', '>=', now())->orderBy('date')->orderBy('start_at')->get() as $meeting)
```

Comment:
please update the `$meeting` variable name here because it might conflict with other `$meeting` variable from your controller.

Reply:
I changed the relevant point from $meeting to $meeting_a .